config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,10 @@
+if Rails.env.production? && Rails.configuration.kubernetes_deployment?
```

Comment:
We have a new redis instance created in the new Cloud Platform environment. We only want to point to it if we are deploying to Kubernetes else template deploy will break